helm-charts/splunk-otel-collector/templates/config/_otel-collector.tpl

```diff
@@ -221,6 +230,9 @@ service:
         - batch
         - filter/logs
         - resource/logs
+        {{- if .Values.logsCollection.containers.useSplunkIncludeAnnotation }}
+        - filter/include_pod_logs
+        {{- end }}
```

Comment:
This filter should go before `resource/logs` to apply the include attribute removal. same in `otelCollector`